pkg/testcase/secretsencrypt.go

```diff
@@ -90,16 +101,16 @@ func waitForHashMatch(cpIP, product string) (string, error) {
 	for i := 1; i <= times; i++ {
 		secretEncryptStatus, errGetStatus = shared.SecretEncryptOps("status", cpIP, product)
 		if errGetStatus != nil {
-			shared.LogLevel("DEBUG", "error getting secret-encryption status. Retry.")
+			shared.LogLevel("debug", "error getting secret-encryption status. Retry.")
 		}
 		if secretEncryptStatus != "" && strings.Contains(secretEncryptStatus, "All hashes match") {
-			shared.LogLevel("DEBUG", "Total sleep time before hashes matched: %d seconds", i*int(defaultTime))
+			shared.LogLevel("debug", "Total sleep time before hashes matched: %d seconds", i*int(defaultTime))
```

Comment:
Let rephrase this message as we don't actually wait initially or if you want to reflect the same then lets add the wait after for loop

Reply:
Yes and actually reflect that it waited for x seconds... coz I see even if you dont wait for x seconds right now it will print that it waited for 10 (default) seconds, which is not true

Reply:
I changed the for loop to start from 0 to < times. that way, if it matched right away - it will matched after 0 seconds. and not 10 seconds default.

Reply:
I am not seeing this change that you have added...

Reply:
https://github.com/rancher/distros-test-framework/pull/194/files#diff-d178807df3cc2529cbf5b801c72ee6a584847d796a2c5693c7785b0e45f9c7f8R104